device/mellanox/x86_64-mlnx_msn4700-r0/Mellanox-SN4700-A96V16/hwsku.json

```diff
@@ -0,0 +1,340 @@
+{
+    "interfaces": {
+        "Ethernet0": {
+            "default_brkout_mode": "4x25G(4)[10G,1G]"
+        },
+        "Ethernet2": {
```

Comment:
Please modify the numbering to Ethernet0, Ethernet1, Ethernet2, Ethernet3

Reply:
Updated

```
"Ethernet0": {
        "index": "1,1,1,1,1,1,1,1",
        "lanes": "0,1,2,3,4,5,6,7",
        "breakout_modes": {
            "1x400G[200G,100G,50G,40G,25G,10G,1G]": [
                "etp1"
            ],
            "2x200G[100G,50G,40G,25G,10G,1G]": [
                "etp1a",
                "etp1b"
            ],
            "4x100G[50G,25G,10G,1G]": [
                "etp1a",
                "etp1b",
                "etp1c",
                "etp1d"
            ],
            "4x25G(4)[10G,1G]": [
                "etp1a",
                "etp1b",
                "etp1c",
                "etp1d"
            ]
        },
        "default_brkout_mode": "4x25G(4)[10G,1G]",
        "Current Breakout Mode": "4x25G(4)[10G,1G]",
        "child ports": "Ethernet0,Ethernet1,Ethernet2,Ethernet3",
        "child port speeds": "25G,25G,25G,25G"
    },
```

```
root@r-leopard-01:/home/admin# config interface breakout Ethernet0 '4x25G(4)[10G,1G]'
Do you want to Breakout the port, continue? [y/N]: y

Running Breakout Mode : 1x400G[200G,100G,50G,40G,25G,10G,1G]
Target Breakout Mode : 4x25G(4)[10G,1G]

Ports to be deleted :
 {
    "Ethernet0": "400000"
}
Ports to be added :
 {
    "Ethernet0": "25000",
    "Ethernet1": "25000",
    "Ethernet2": "25000",
    "Ethernet3": "25000"
}

After running Logic to limit the impact

Final list of ports to be deleted :
 {
    "Ethernet0": "400000"
}
Final list of ports to be added :
 {
    "Ethernet0": "25000",
    "Ethernet1": "25000",
    "Ethernet2": "25000",
    "Ethernet3": "25000"
}
sonic_yang(6):Note: Below table(s) have no YANG models: SNMP, SNMP_COMMUNITY
Below Config can not be verified, It may cause harm to the system
 {}
Do you wish to Continue? [y/N]: y
Breakout process got successfully completed.
Please note loaded setting will be lost after system reboot. To preserve setting, run `config save`.

```